### PR TITLE
Doc: extension for terminal output

### PR DESCRIPTION
**Pull request by ru-fu**
Draft PR for review - before merging, I'll move the extension out to our extension package.

### CODE REVIEW

File: doc/howto/network_zones.md [hunk 1]
```diff
@@ -53,7 +53,11 @@ You can check the records that are generated with your zone setup with the `dig`
 This assumes that `core.dns_address` was set to `<DNS_server_IP>:<DNS_server_PORT>`.
 For example, running `dig @<DNS_server_IP> -p <DNS_server_PORT> axfr lxd.example.net` might give the following output:
 
-```bash
+```{terminal}
+:input: dig @192.0.2.200 -p 1053 axfr lxd.example.net
```
Comment (ru-fu) [inline doc/howto/network_zones.md:57]: Does this example IP make sense, or should it be one that appears in the output?

Comment (tomponline) [reply doc/howto/network_zones.md:57]: I'm not sure I understand the question? What output?

Comment (tomponline) [reply doc/howto/network_zones.md:57]: Yes that makes sense to me.

**Comment by ru-fu** [comment]
This depends on https://github.com/canonical/lxd-sphinx-extensions/pull/10 (see last commit), but is otherwise ready for review.

**Pull request merged**